[Bug]: [Nightly]Load collection failed reporting "DescribeCollection failed: can't find collection:xxx"

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master(2bafdf8)
- Deployment mode(standalone or cluster):standalone
- SDK version(e.g. pymilvus v2.0.0rc2):2.2.0.dev26
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

These cases failed:
```
[2022-09-05T15:18:59.133Z] [gw0] [  9%] FAILED testcases/test_collection.py::TestLoadCollection::test_load_collection_not_existed
[2022-09-05T15:19:02.088Z] [gw1] [  9%] FAILED testcases/test_collection.py::TestLoadCollection::test_release_collection_not_existed
[2022-09-05T15:19:25.911Z] [gw4] [ 10%] FAILED testcases/test_collection.py::TestLoadCollection::test_load_release_collection
```

### Expected Behavior

PASSED

### Steps To Reproduce

```markdown
nightly:
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/80/pipeline/123
```


### Milvus Log

1. collection name: test_4uTnR4da
2. log: [artifacts-milvus-standalone-nightly-80-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Milvus%20Nightly%20CI/job/master/80/artifact/artifacts-milvus-standalone-nightly-80-pymilvus-e2e-logs.tar.gz)
3. error message:
```
[2022-09-05T16:57:58.106Z] [2022-09-05 15:18:58 - ERROR - pymilvus.decorators]: RPC error: [load_collection], <MilvusException: (code=1, message=DescribeCollection failed: can't find collection: 435785768457412872)>, <Time:{'RPC start': '2022-09-05 15:18:58.217056', 'RPC error': '2022-09-05 15:18:58.222972'}> (decorators.py:100)
[2022-09-05T16:57:58.106Z] [2022-09-05 15:18:58 - ERROR - ci_test]: Traceback (most recent call last):
[2022-09-05T16:57:58.106Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 26, in inner_wrapper
[2022-09-05T16:57:58.106Z]     res = func(*args, **_kwargs)
[2022-09-05T16:57:58.106Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 57, in api_request
[2022-09-05T16:57:58.106Z]     return func(*arg, **kwargs)
[2022-09-05T16:57:58.106Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/orm/collection.py", line 474, in load
[2022-09-05T16:57:58.106Z]     conn.load_collection(self._name, replica_number=replica_number, timeout=timeout, **kwargs)
[2022-09-05T16:57:58.106Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 101, in handler
[2022-09-05T16:57:58.106Z]     raise e
[2022-09-05T16:57:58.106Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 97, in handler
[2022-09-05T16:57:58.106Z]     return func(*args, **kwargs)
[2022-09-05T16:57:58.106Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 127, in handler
[2022-09-05T16:57:58.106Z]     ret = func(self, *args, **kwargs)
[2022-09-05T16:57:58.107Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 79, in handler
[2022-09-05T16:57:58.107Z]     raise e
[2022-09-05T16:57:58.107Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 50, in handler
[2022-09-05T16:57:58.107Z]     return func(self, *args, **kwargs)
[2022-09-05T16:57:58.107Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 697, in load_collection
[2022-09-05T16:57:58.107Z]     raise MilvusException(response.error_code, response.reason)
[2022-09-05T16:57:58.107Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=1, message=DescribeCollection failed: can't find collection: 435785768457412872)>
[2022-09-05T16:57:58.107Z]  (api_request.py:39)
[2022-09-05T16:57:58.107Z] [2022-09-05 15:18:58 - ERROR - ci_test]: (api_response) : <MilvusException: (code=1, message=DescribeCollection failed: can't find collection: 435785768457412872)> (api_request.py:40)
```

### Anything else?

_No response_

This error occurs frequently.
https://jenkins.milvus.io:18080/job/Nightly-Debug/job/test-binbin-nightly-debug/4/
1. milvus: cbae252   pymilvus: 2.2.0.dev26
2. error cases: 
```
[2022-09-07T19:07:40.374Z] [gw0] [  9%] FAILED testcases/test_collection.py::TestLoadCollection::test_load_collection_not_existed
[2022-09-07T19:07:42.576Z] [gw1] [  9%] FAILED testcases/test_collection.py::TestLoadCollection::test_release_collection_not_existed
[2022-09-07T19:08:02.875Z] [gw5] [ 10%] FAILED testcases/test_collection.py::TestLoadCollection::test_load_release_collection
```
3. log: [artifacts-milvus-standalone-nightly-4-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Nightly-Debug/job/test-binbin-nightly-debug/4/artifact/artifacts-milvus-standalone-nightly-4-pymilvus-e2e-logs.tar.gz)
4. error message:
```
[2022-09-07T21:03:39.619Z] [2022-09-07 19:07:41 - ERROR - pymilvus.decorators]: RPC error: [release_collection], <MilvusException: (code=1, message=DescribeCollection failed: can't find collection: 435834723110691456)>, <Time:{'RPC start': '2022-09-07 19:07:41.175818', 'RPC error': '2022-09-07 19:07:41.561366'}> (decorators.py:100)
[2022-09-07T21:03:39.619Z] [2022-09-07 19:07:41 - ERROR - ci_test]: Traceback (most recent call last):
[2022-09-07T21:03:39.619Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 26, in inner_wrapper
[2022-09-07T21:03:39.619Z]     res = func(*args, **_kwargs)
[2022-09-07T21:03:39.619Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 57, in api_request
[2022-09-07T21:03:39.619Z]     return func(*arg, **kwargs)
[2022-09-07T21:03:39.619Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/orm/collection.py", line 501, in release
[2022-09-07T21:03:39.619Z]     conn.release_collection(self._name, timeout=timeout, **kwargs)
[2022-09-07T21:03:39.619Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 101, in handler
[2022-09-07T21:03:39.619Z]     raise e
[2022-09-07T21:03:39.619Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 97, in handler
[2022-09-07T21:03:39.619Z]     return func(*args, **kwargs)
[2022-09-07T21:03:39.619Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 127, in handler
[2022-09-07T21:03:39.619Z]     ret = func(self, *args, **kwargs)
[2022-09-07T21:03:39.619Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 79, in handler
[2022-09-07T21:03:39.619Z]     raise e
[2022-09-07T21:03:39.619Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 50, in handler
[2022-09-07T21:03:39.619Z]     return func(self, *args, **kwargs)
[2022-09-07T21:03:39.619Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 752, in release_collection
[2022-09-07T21:03:39.619Z]     raise MilvusException(response.error_code, response.reason)
[2022-09-07T21:03:39.619Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=1, message=DescribeCollection failed: can't find collection: 435834723110691456)>
[2022-09-07T21:03:39.619Z]  (api_request.py:39)
[2022-09-07T21:03:39.619Z] [2022-09-07 19:07:41 - ERROR - ci_test]: (api_response) : <MilvusException: (code=1, message=DescribeCollection failed: can't find collection: 435834723110691456)> (api_request.py:40)
```

--- a/tests/python_client/testcases/test_collection.py
+++ b/tests/python_client/testcases/test_collection.py
@@ -2064,7 +2064,7 @@ class TestLoadCollection(TestcaseBase):
         c_name = cf.gen_unique_str()
         collection_wr = self.init_collection_wrap(name=c_name)
         collection_wr.drop()
-        error = {ct.err_code: 0,
+        error = {ct.err_code: 1,
                  ct.err_msg: "DescribeCollection failed: can't find collection: %s" % c_name}
         collection_wr.load(check_task=CheckTasks.err_res, check_items=error)
 
@@ -2079,7 +2079,7 @@ class TestLoadCollection(TestcaseBase):
         c_name = cf.gen_unique_str()
         collection_wr = self.init_collection_wrap(name=c_name)
         collection_wr.drop()
-        error = {ct.err_code: 0,
+        error = {ct.err_code: 1,
                  ct.err_msg: "DescribeCollection failed: can't find collection: %s" % c_name}
         collection_wr.release(check_task=CheckTasks.err_res, check_items=error)
 
@@ -2141,7 +2141,7 @@ class TestLoadCollection(TestcaseBase):
         collection_wr.load()
         collection_wr.release()
         collection_wr.drop()
-        error = {ct.err_code: 0,
+        error = {ct.err_code: 1,
                  ct.err_msg: "DescribeCollection failed: can't find collection: %s" % c_name}
         collection_wr.load(check_task=CheckTasks.err_res, check_items=error)
         collection_wr.release(check_task=CheckTasks.err_res, check_items=error)

